onnxconverter_common/onnx_ops.py

```diff
@@ -563,7 +563,7 @@ def apply_inverse(scope, input_name, output_name, container, operator_name=None)
     container.add_node('Inverse', input_name, output_name, name=name, op_version=op_version)
 
 
-def apply_leaky_relu(scope, input_name, output_name, container, operator_name=None, alpha=None):
```

Comment:
alpha : float (default is 0.01)
Coefficient of leakage default to 0.01.

Reply:
See [here](https://www.tensorflow.org/api_docs/python/tf/nn/leaky_relu), it is 0.2.

Reply:
synced offline, use onnx spec 0.01